dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -68,11 +68,13 @@ public FileAccessIO(T dvObject, DataAccessRequest req) {
 
     // "Direct" File Access IO, opened on a physical file not associated with
     // a specific DvObject
-    public FileAccessIO(String storageLocation) {
+    public FileAccessIO(String storageLocation, String filesDirectory) {
```

Comment:
This is only one of multiple constructors. What about the rest?

Reply:
we only use FilesAccessIO.filesRootDirectory on objects created with (String,String) constructor. 
IMO there's no need to add this parameter to other constructors aswell.

Reply:
This is not true.
For example if we create `FileAccessIO` with different constructor and then on created object run `listAuxObjects` we will then go inside listCachedFiles where filesRootDirectory will be used